src/workspacer.FSharpConfig/Extensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace workspacer.FSharpConfig
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+
+    public static class Extensions
+    {
+        public static Assembly LoadFile(this AppDomain appDomain, string filename) {
+            FileStream fs = new FileStream(filename, FileMode.Open);
+            byte[] buffer = new byte[(int) fs.Length];
+            fs.Read(buffer, 0, buffer.Length);
+            fs.Close();
+
+            return appDomain.Load(buffer);
+        }
+
+        public static AppDomain FileResolver(this AppDomain resoleAppDomain, IEnumerable<string> dllFiles)
+        {
+            var references = dllFiles.ToDictionary(x => AssemblyName.GetAssemblyName(x).FullName);
+
+            Assembly Resolver(object sender, ResolveEventArgs args)
+            {
+                AppDomain domain = (AppDomain) sender!;
+
+                var file = references[args.Name];
+                return LoadFile(domain, file);
+            }
```

Comment:
I think declaring this event-handler function here instead of directly attaching it to the event makes it much more confusing to read.